components/authn-service/tokens/tokens_test.go

```diff
@@ -15,11 +15,13 @@ import (
 	"github.com/stretchr/testify/require"
 	"go.uber.org/zap"
 
+	"github.com/chef/automate/components/authn-service/constants"
```

Comment:
Note to reviewer:

I cleaned up this file in anticipation of adding tests that ended up in `pg_test`